cmd/harmony/config_migrations.go

```diff
@@ -103,6 +103,10 @@ func init() {
 			confTree.Set("P2P.IP", defaultConfig.P2P.IP)
 		}
 
+		if confTree.Get("P2P.Concurrency") == nil {
```

Comment:
This is a new field, and you need to update the config version. Also it is recommended that you rebase on top of https://github.com/harmony-one/harmony/pull/3773

Reply:
need to wait the PR you mentioed merged, and I can rebase on the top of the PR.  Otherwise this PR may still be changed